backend/accounts/account.go

```diff
@@ -32,6 +32,14 @@ type AddressList struct {
 	Addresses  []Address
 }
 
+// AccountBalance includes the total and incoming balance of an account.
+type AccountBalance struct {
```

Comment:
there is already a `Balance` struct (in balance.go). making this ultra confusing.

Maybe move this to balance.go too and name it `FormattedAccountBalance`.